packages/common/test/AncillaryDataParser.js

```diff
@@ -0,0 +1,150 @@
+// Script to test
+const AncillaryDataParser = require("../dist/AncillaryDataParser");
+
+contract("AncillaryDataParser.js", function () {
+  describe("parseAncillaryData", function () {
+    it("parses simple key-value data correctly", async function () {
+      const data = web3.utils.utf8ToHex("key:value");
+      const expectedObject = { key: "value" };
+      const parsedData = AncillaryDataParser.parseAncillaryData(data);
+      assert.equal(JSON.stringify(parsedData).toLowerCase(), JSON.stringify(expectedObject).toLowerCase());
```

Comment:
why do you need to cast this to lowercase?

Reply:
I don't remember why these conversions were in there originally but I removed them and everything was fine.